app/models/vm_or_template.rb

```diff
@@ -1522,7 +1522,7 @@ def changed_vm_value?(options)
   #
 
   virtual_delegate :allocated_disk_storage, :used_disk_storage, :provisioned_storage,
-                   :to => :hardware, :allow_nil => true, :uses => {:hardware => :disks}
+                   :to => :hardware, :allow_nil => true, :default => 0, :uses => {:hardware => :disks}
```

Comment:
yea, the `:default => 0` for `:provisioned_storage` is right.

But `:allocated_disk_storage` `and `:used_disk_storage` clearly want a default of `nil`.
(not saying this is right, just saying